src/main/java/zmaster587/libVulpes/tile/multiblock/TileMultiBlock.java

```diff
@@ -200,12 +198,15 @@ public void deconstructMultiBlock(World world, BlockPos destroyedPos, boolean bl
 					int globalZ = pos.getZ() - (x - offset.x)*front.getFrontOffsetX()  - (z-offset.z)*front.getFrontOffsetZ();
 
 
+					TileEntity tile = world.getTileEntity(new BlockPos(globalX, globalY, globalZ));
 					//This block is being broken anyway so don't bother
-					if(blockBroken && globalX == destroyedPos.getX() &&
-							globalY == destroyedPos.getY() &&
-							globalZ == destroyedPos.getZ())
+					if(blockBroken && globalX == destroyedPos.getX() && globalY == destroyedPos.getY() && globalZ == destroyedPos.getZ()) {
```

Comment:
Should stay on multiple lines for readability, however the operator should come first, not at the end like the previous code did.

```suggestion
					if(blockBroken 
					&& globalX == destroyedPos.getX() 
					&& globalY == destroyedPos.getY() 
					&& globalZ == destroyedPos.getZ()) {
```